scripts/pin_the_slow_way.py

```diff
@@ -41,7 +41,7 @@
           'glib': 'glib 2.51.*',  # 2.50.2
           'gmp': 'gmp >=5.0.1,<7', # 6.1.0
           'harfbuzz': 'harfbuzz 1.3.*',  # 0.9.39
-          'hdf5': 'hdf5 1.8.17|1.8.17.*',  # 1.8.17
+          'hdf5': 'hdf5 1.8.18|1.8.18.*',  # 1.8.17
```

Comment:
@jjhelmus can you confirm if `defaults` version is still `1.8.17`?

(It is probably safe for `conda-forge` to move to `1.8.18` b/c we do have all the packages that uses `hdf5` packaged in `conda-forge`.)